docker/README.md

```diff
@@ -217,43 +222,40 @@ docker build -f docker/Dockerfile --target=build -t zwavejs/zwavejs2mqtt_build .
 
 ```
 
-## SSH inside container
+## Building a container using Dockerfile.contrib
 
-```bash
-docker run --rm -p 8091:8091 --device=/dev/ttyACM0 -it --mount source=zwavejs2mqtt,target=/usr/src/app zwavejs/zwavejs2mqtt:latest sh
-```
+You can also build a custom docker image using any mix of branches or repositories (such as your own), simply run the following series of commands, indicating the name of the branches you wish to build for node-zwave-js (ZWJ_BRANCH) and zwavejs2mqtt (Z2M_BRANCH) and the resulting docker image name (e.g. zwavejs2mqtt):
```

Comment:
Should we be explicit here and mention that `ZWJ_BRANCH=master` needs to be replaced with `ZWJ_BRANCH=<your-branch-name>` (same for Z2M_...)

Reply:
It is mentioned below